README.md

```diff
@@ -1,138 +1,221 @@
-# rapids-env-generator
+# rapids-dependency-file-generator
 
-`rapids-env-generator` is a Python package that generates multiple `conda` environment files from a single YAML file. When installed via `pip`, it makes the `rapids-env-generator` CLI command available which is responsible for parsing an `envs.yaml` configuration file and generating the appropriate `conda` environment files.
+`rapids-dependency-file-generator` is a Python CLI tool that generates `conda` environment files and `requirements.txt` files from a single YAML file, typically named `dependencies.yaml`. When installed, it makes the `rapids-dep-file-generator` CLI command available which is responsible for parsing a `dependencies.yaml` configuration file and generating the appropriate `conda` environment and `requirements.txt` dependency files.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [`dependencies.yaml` Format](#dependenciesyaml-format)
+  - [`files` key](#files-key)
+  - [`channels` key](#channels-key)
+  - [`dependencies` key](#dependencies-key)
+- [How Dependency Lists Are Merged](#how-dependency-lists-are-merged)
+- [Additional CLI Notes](#additional-cli-notes)
+
+## Installation
+
+`rapids-dependency-file-generator` is available on PyPI. To install, run:
+
+```sh
+pip install rapids-dependency-file-generator
```

Comment:
It's awkward that the package name and the CLI command are not the same. I'd vote for `rapids-dependency-file-generator` everywhere. I was also okay with the previous name `rapids-env-generator` (I don't think it sounds conda-specific).

Reply:
Sure, I can update that. I didn't like how verbose `rapids-dependency-file-generator` was, but I agree that consistency is probably better.

Reply:
It is verbose, but it's just a tab-complete away in most shells, and the verbosity won't hurt if it's called in a script. The arbitrariness of the abbreviation was also questionable - `rapids-dfg` or `rapids-dep-gen` would have been equally okay, but I prefer the full name.